VeganVirus/VeganVirus/ClipBoardAction.cpp

```diff
@@ -0,0 +1,60 @@
+#include "ClipBoardAction.h"
+
+
+ClipBoardAction::ClipBoardAction(double req) : Action(req, CLIP_BOARD_ICON_PATH)
```

Comment:
Hii, there is unused parmater in the function look if you relly need this. or wait for alon CR.